package.json

```diff
@@ -1,22 +1,24 @@
-{
-    "name": "fellers-bot",
-    "version": "1.0.0",
-    "description": "The fellers first bot",
-    "main": "bot.js",
-    "author": "Calvin Chhour",
-    "dependencies": {
-        "discord.js": "^11.5.1",
-        "mongodb": "^3.3.0",
-        "winston": "^3.2.1"
-    },
-    "scripts": {
-        "lint": "eslint src/",
-        "lint-fix": "eslint src/ --fix",

```

Comment:
Don't remove this.

Reply:
I don't remember removing it, how would I remove it by accident?

Reply:
Probably just a rebase issue. Easy fix tho xd

Reply:
Probably just a rebase issue. Easy fix tho xd